step-templates/ssis-deploy-ispac-with-enviroment.json

```diff
@@ -0,0 +1,73 @@
+  },
+  "Parameters": [
+    {
+      "Id": "4d49a1f7-f7bf-40e2-b845-8c9c8e4deda9",
+      "Name": "NugetPackageStepName",
```

Comment:
These names should be prefixed so that they don't clash with project (or library) level variables. I suggest `IspacsEnv`.

Reply:
Update with the Prefix